velox/functions/prestosql/Comparisons.cpp

```diff
@@ -112,11 +113,14 @@ struct SimdComparator {
     auto resultVector = result->asUnchecked<FlatVector<bool>>();
     auto rawResult = resultVector->mutableRawValues<uint8_t>();
 
+    constexpr bool isDecimal =
+        (std::is_same_v<T, UnscaledShortDecimal> ||
```

Comment:
These changes will be overwritten with Bigint and huge INR representations after this Pr is merged https://github.com/facebookincubator/velox/pull/4434

Reply:
We probably should merge https://github.com/facebookincubator/velox/pull/4434 first then?

Reply:
We need this PR to be merged before #4434 since there is no provision for decimal scalar functions there.
As @karteekmurthys said, we will be replacing UnscaledShortDecimal with int64_t and UnscaledLongDecimal with int128_t. So we can keep this as is for now.